<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>card</class>
    <widget class="QWidget" name="card">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>361</width>
                <height>300</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>300</width>
                <height>300</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>16777215</width>
                <height>16777215</height>
            </size>
        </property>
        <property name="sizeIncrement">
            <size>
                <width>0</width>
                <height>0</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
            <string notr="true">#card{
                background-color: rgba(30, 30, 30,180);
                border-radius:10px;
                text-align: :center;
                }

                QWidget{
                background-color: rgba(50, 50, 50,0) !important;
                }

                QPushButton{
                background-color: rgba(20, 20, 20,0) !important;
                }

            </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="sizeConstraint">
                <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <property name="leftMargin">
                <number>5</number>
            </property>
            <property name="topMargin">
                <number>5</number>
            </property>
            <property name="rightMargin">
                <number>5</number>
            </property>
            <property name="bottomMargin">
                <number>5</number>
            </property>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <property name="leftMargin">
                        <number>30</number>
                    </property>
                    <property name="rightMargin">
                        <number>0</number>
                    </property>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                        <widget class="QLabel" name="title">
                            <property name="text">
                                <string>TextLabel</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="delete_op">
                            <property name="minimumSize">
                                <size>
                                    <width>30</width>
                                    <height>30</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>30</width>
                                    <height>30</height>
                                </size>
                            </property>
                            <property name="layoutDirection">
                                <enum>Qt::LeftToRight</enum>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                            <property name="flat">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QScrollArea" name="scrollArea">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                            <horstretch>1</horstretch>
                            <verstretch>1</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Plain</enum>
                    </property>
                    <property name="lineWidth">
                        <number>0</number>
                    </property>
                    <property name="horizontalScrollBarPolicy">
                        <enum>Qt::ScrollBarAlwaysOff</enum>
                    </property>
                    <property name="sizeAdjustPolicy">
                        <enum>QAbstractScrollArea::AdjustIgnored</enum>
                    </property>
                    <property name="widgetResizable">
                        <bool>true</bool>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <widget class="QWidget" name="progress_container">
                        <property name="geometry">
                            <rect>
                                <x>0</x>
                                <y>0</y>
                                <width>351</width>
                                <height>258</height>
                            </rect>
                        </property>
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <property name="sizeConstraint">
                                <enum>QLayout::SetDefaultConstraint</enum>
                            </property>
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
